packages/api/src/test-server.ts

```diff
@@ -1,14 +1,15 @@
 /**
  * This file is imported from all the integration tests. It boots up a server based on the provided argv.
  */
-
-import argParser from "./parse-cli";
-import makeApp from "./index";
 import fs from "fs-extra";
-import uuid from "uuid/v4";
+import { v4 as uuid } from "uuid";
 import path from "path";
 import os from "os";
-import fetch from "isomorphic-fetch";
+import "isomorphic-fetch";
```

Comment:
I know this works but dumping stuff on the global scope is a little frightening. Want to just use `node-fetch` instead?

Reply:
Switched to `node-fetch`!

Answering the comment though: It actually doesn't make a difference which way we import the library, as it always messes with the global scope when imported. So I had switched to not use the default export as I thought it was a bit misleading, using the value it sets on the global scope instead. I guess this import worked like a "blank identifier import" in go.

In fact:
 - `global.fetch` and `require("isomorphic-fetch")` are the exact same value as they do [`global.fetch = module.exports;` here](https://github.com/matthew-andrews/isomorphic-fetch/blob/master/fetch-npm-node.js#L12)
 - Also learned that `isomorphic-fetch` basically exposes `node-fetch` on the global scope, doing like 1 `//`=>`https://` conversion before calling `node-fetch`. So the type hack I made doesn't even feel that much like a hack anymore 😆 (but it's much better to just use `node-fetch` directly anyway)

Reply:
Btw, the whole typing problem with `isomorphic-fetch` is that [their DefinitelyTyped package](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/isomorphic-fetch/index.d.ts#L8) assumes that it's going to run on a browser and have the `fetch` type in the global scope. So we then also had to transpile our node app with the `dom` module, which could lead to programming errors if you use something from the browser not available in node. Kinda non-sense if you ask me, given the purpose of this isomorphic package, but I'm also not a TS expert to know how to fix it easily on their repository. So let's just stick with `node-fetch` for now 😬